index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+      integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Portfolio Setup and Mobile first</title>
+  </head>
+  <body>
+    <div class="logo-menu">
+      <div class="logo" onclick="window.location.href='index.html'">
+        <p class="logo">My Logo</p>
+      </div>
+      <i class="fa fa-bars"></i>
+    </div>
+    <main>
+      <div class="main-container">
+        <div class="title">
+          <h2>I’m Mariane Glad to see you!</h2>
```

Comment:
- [ ] Kindly I recommend you to use `h1` tag for "Main Title(I’m Mariane Glad to see you!)",  This way will be semantically correct and the screen readers and other technologies will thank you,[Check this link](https://www.reliablesoft.net/h1-tag/) for more info..